model-optimizer/mo/main.py

```diff
@@ -297,6 +305,9 @@ def main(cli_parser: argparse.ArgumentParser, framework: str):
         ret_code = driver(argv)
         if ov_update_message:
             print(ov_update_message)
+        telemetry.send_event('mo', 'conversion_result', 'success')
+        telemetry.end_session()
+        telemetry.force_shutdown(1.0)
```

Comment:
Why do you need a special API for `force_shutdown`? I see that it is used in conjunction with `end_session` every time. Maybe it is worth to simplify the API and call this method internally.

Reply:
The application itself defines what is "session". So you may define as "session" some particular step of your application and in this case you have multiple sessions between start of the application and end of it. So we should not call `force_shutdown` internally.